package.json

```diff
@@ -32,6 +32,8 @@
     "lint:specs:fix": "npx tslint --project . -t stylish --fix -c tslint.specs.js 'src/**/*.{spec,e2e}.{ts,tsx}'",
     "lint:src": "npx tslint --project . -t stylish -c tslint.js 'src/**/*.{ts,tsx}'",
     "lint:src:fix": "npx tslint --project . -t stylish --fix -c tslint.js 'src/**/*.{ts,tsx}'",
+    "lint:scss": "prettier -c **/*.scss",
+    "lint:scss:fix": "prettier --write **/*.scss",
```

Comment:
When running this, it will rewrite scss files inside the `.docz` folder too, but if I change to `src/**/*.scss` it doesn't run on the files in the `src/components/` folder for some reason I really can't understand.

@jgroth Do you have any idea?

Anyone else?

Reply:
have you tried to ignore the `.docz` path/ add a `.prettierignore`? https://prettier.io/docs/en/cli.html#--ignore-path

Reply:
Ah, of course! 🙏

Reply:
Adding `--ignore-path ./.gitignore` worked, and seems like quite a silvery bullet to me 😁